pkg/yamltemplate/template_test.go

```diff
@@ -24,12 +24,22 @@ import (
 
 var (
 	// Example usage:
-	//   ./hack/test.sh -run TestYAMLTemplate TestYAMLTemplate.filetest=yield-def.yml
-	selectedFileTestPath = kvArg("TestYAMLTemplate.filetest")
-	showTemplateCode     = kvArg("TestYAMLTemplate.code") // eg t|...
-	showErrs             = kvArg("TestYAMLTemplate.errs") // eg t|...
+	//   ./hack/test-all.sh -v -run TestYAMLTemplate/filetest/if.tpltest
+	//   ./hack/test-all.sh -v -run TestYAMLTemplate/filetest/if.tpltest TestYAMLTemplate.code=true
```

Comment:
Optional: since the path here works for `../filetest/..` _and_ `../filetests/..` we can change this to `filetests` to have it match the directory structure exactly and reduce confusion in the slight naming difference. 

```suggestion
	//   ./hack/test-all.sh -v -run TestYAMLTemplate/filetests/if.tpltest
	//   ./hack/test-all.sh -v -run TestYAMLTemplate/filetests/if.tpltest TestYAMLTemplate.code=true
```

Reply:
Nice catch; thank you!

Fixed.